health/health_log.c

```diff
@@ -145,6 +146,10 @@ inline void health_alarm_log_save(RRDHOST *host, ALARM_ENTRY *ae) {
                             , ae->new_value
                             , ae->old_value
                             , (uint64_t)ae->last_repeat
+
```

Comment:
Please, remove this blank line.

Reply:
Oks with https://github.com/netdata/netdata/pull/10850/commits/0918653d9dd04ac94ef0b8968bc10db6e7a61cce